src/content/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase.mdx

```diff
@@ -48,12 +48,12 @@ After setting up New Relic Infrastructure Amazon integrations, your usage of the
     id="usage"
     title="Review API usage"
   >
-    To review the API usage for New Relic Infrastructure integrations with Amazon AWS:
+    To review the API usage for New Relic Infrastructure integrations with AWS:
 
     1. Go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > AWS > Account status dashboard**.
     2. Review the dashboard.
 
-    The dashboard includes a chart with your account's Amazon AWS API call count for the last month as well as the CloudWatch API calls (per AWS resource) for the last day. This information is the API usage for New Relic only. It does not include other AWS API or CloudWatch usage that may occur.
+    The dashboard includes a chart with your account's  API call count for the last month as well as the CloudWatch API calls (per AWS resource) for the last day. This information is the API usage for New Relic only. It does not include other AWS API or CloudWatch usage that may occur.
```

Comment:
looks like extra space between "account's" and "API"

Reply:
there was! you rock!